### PR TITLE
moved a warning message from appium into appium-instruments

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

**Comment by jlipps** [comment]
unit tests failed for this

**Comment by Jonahss** [comment]
How was I able to commit? Are the unit tests no running on pre-commit hooks?

**Comment by Jonahss** [comment]
ok fixed. and my next update to this project will even remove the null check.

**Comment by jlipps** [comment]
(for posterity) pre-commit hooks are something you have to set up yourself, so we didn't include them in this project

File: lib/instruments.js [hunk 1]
```diff
@@ -69,6 +69,13 @@ var Instruments = function (opts) {
   this.tmpDir = opts.tmpDir || '/tmp/appium-instruments';
   this.traceDir = opts.traceDir || this.tmpDir;
   this.gotFBSOpenApplicationError = false;
+
+  if (this.xcodeVersion && this.xcodeVersion.slice(0,3) === '6.0' && this.withoutDelay) {
```
Comment (jlipps) [inline lib/instruments.js]: `slice(0,3)` should be `slice(0, 3)`

**Comment by Jonahss** [comment]
Fixed spacing.

**Comment by Jonahss** [comment]
spaced fixing

**Comment by jlipps** [comment]
ok, i merged this and published appium-instruments 1.5.5. i'll let you get it into appium

**Pull request merged**